.github/workflows/release-vscode-extension.yml

```diff
@@ -36,15 +36,26 @@ jobs:
         run: |
           rustup target add x86_64-unknown-linux-musl
           cargo build --release --manifest-path cargo-concordium/Cargo.toml --target x86_64-unknown-linux-musl
-      - name: Build on other platforms
-        if: matrix.platfrom != 'ubuntu-latest'
+      - name: Build on windows
+        if: matrix.platfrom == 'windows-latest'
         run: cargo build --release --manifest-path cargo-concordium/Cargo.toml
+      - name: Build on macos
+        if: matrix.platfrom == 'macos-latest'
+        run: |
+          # Build x86_64
+          cargo build --release --manifest-path cargo-concordium/Cargo.toml
+          # Build ARM64 and rename it.
+          rustup target add aarch64-apple-darwin
+          cargo build --release --manifest-path cargo-concordium/Cargo.toml --target aarch64-apple-darwin
+          mv cargo-concordium/target/release/aarch64-apple-darwin/cargo-concordium cargo-concordium/target/release/aarch64-apple-darwin/cargo-concordium-aarch64
```

Comment:
It seems redundant to specify architecture in both the directory and filename?

```suggestion
```

Alternatively move the file to 
```
cargo-concordium/target/release/cargo-concordium-aarch64
```